components/ui/svg.tsx

```diff
@@ -60,7 +60,7 @@ export class SvgImage {
   }
 
   static kebabIcon() {
-    return <KebabIcon />;
+    return <KebabIcon {...testIDProps('KebabIcon')} />;
```

Comment:
send testId to this kebabIcon() function as it can be used at many places with different context